src/Client.php

```diff
@@ -75,14 +75,21 @@ public function getUser(): User
      * @param SenderAddress|int|null $senderAddress The sender address to ship from. Methods available to all of your
      * account's sender addresses will be retrieved when null.
      * @param bool $returnMethodsOnly When true, methods for making a return are returned instead.
+     * @param int from_postal_code
+     * @param int to_postal_code
+     * @param int to_country iso2 code
      * @return ShippingMethod[]
      * @throws SendcloudClientException
      * @see https://sendcloud.dev/docs/shipping/shipping-methods/
      */
     public function getShippingMethods(
         ?int $servicePointId = null,
         SenderAddress|int|null $senderAddress = null,
-        bool $returnMethodsOnly = false
+        bool $returnMethodsOnly = false,
+        ?int $from_postal_code =null,
+        ?int $to_postal_code =null,
+        ?string $to_country =null,
```

Comment:
I know it would be better to sync up the naming with Sendcloud's, but consistency with existing code is more important in this case. Let's rename these to use lowerCamelCase like the other arguments!